docs/docs/1.2.0/policies/external-services.md

```diff
@@ -68,27 +82,32 @@ networking:
+* With the real name and port, in this case `curl httpbin.org:443`. This approach works only with [the data plane proxy builtin DNS](../networking/dns/#data-plane-proxy-built-in-dns) name resolution.
 
-Although the external service is HTTPS, it's consumed as plain HTTP. This is possible because of `networking.tls.enbaled=true`.
-To access the service over TLS, set the service protocol to `kuma.io/protocol: tcp` and `networking.tls.enbaled=false`, or else omit it entirely.
+Although the external service is HTTPS, it's consumed as plain HTTP. This is possible because when `networking.tls.enabled` is set to `true` then Envoy is responsible for originating and verifying TLS.
+
+To consume the service using HTTPS, set the service protocol to `kuma.io/protocol: tcp` and `networking.tls.enabled=false`. This way application itself is responsible for originating and verifying TLS and Envoy is just passing the connection to a proper destination.
+
+The first approach has an advantage that we can apply HTTP based policies, because Envoy is aware of HTTP protocol and can apply request modifications before the request is encrypted. Additionally, we can modify TLS certificates without restarting applications.
 
 ### Available policy fields
 
- * `tags` the external service can include an arbitrary number of tags, where `kuma.io/service` is mandatory. The special `kuma.io/protocol` tag is also taken into account and supports the standard Kuma protocol values. It designates the specific protocol for the service.
- * ` networking` describes the networking configuration of the external service 
-   * `address` is the address where the external service can be reached.
-   * `tls` is the section to configure the TLS originator when consuming the external service
-     * `enabled` turns on and off the TLS origination. Defaults to `true`
-     * `caCert` the CA certificate for the external service TLS verification
-     * `clientCert` the client certificate for mTLS
-     * `clientKey` the client key for mTLS
- 
+* `tags` the external service can include an arbitrary number of tags, where `kuma.io/service` is mandatory. The special `kuma.io/protocol` tag is also taken into account and supports the standard Kuma protocol values. It designates the specific protocol for the service.
+* ` networking` describes the networking configuration of the external service
+    * `address` is the address where the external service can be reached.
+    * `tls` is the section to configure the TLS originator when consuming the external service
+        * `enabled` turns on and off the TLS origination.
+        * `allowRenegotiation` turns on and off TLS renegotiation. It's not recommended enabling this for [security reasons](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/tls.proto).
+          However, some servers require this setting to fetch client certificate after TLS handshake. TLS renegotiation is not available in TLS v1.3.
```

Comment:
```suggestion
However, some servers require this setting to fetch client the certificate after the TLS handshake. TLS renegotiation is not available in TLS v1.3.
```

Ignore the lack of leading whitespace in the suggestion ... chrome won't let me paste whitespace in there :shrug: